README.md

```diff
@@ -103,6 +105,18 @@ export default () => (
 </tbody>
 </table>
 
+
+### `<LoadsProvider>`
+
+#### Props
+
+<table>
+<thead><tr><th>Prop</th><th>Type</th><th>Default value</th><th>Description</th></tr></thead>
+<tbody>
+  <tr><td>  storagePrefix </td><td><code>string</code></td><td>react_loads_</td><td>The prefix to use when storing items with this provider. It is recommended you set this to the name of your app or module to avoid namespace collisions in storage.</td></tr>
```

Comment:
I like this prop! A bit of a nit, I think `react-loads.` might be a better prefix?